numpy/core/src/multiarray/einsum_sumprod.c.src

```diff
@@ -94,6 +94,58 @@
  *             0*3#
  */
 
+#if !@complex@
+static NPY_GCC_OPT_3 @temptype@ @name@_sum_of_arr(@type@ **data, npy_intp count)
```

Comment:
```suggestion
static NPY_GCC_OPT_3 @temptype@ @name@_sum_of_arr(@type@ *data, npy_intp count)
```
Why do you passing the value of the pointer address? I see no need for it.

Reply:
you are right, passing the pointer address would be like gild the lily.